### PR TITLE
Add script for generating manifests

### PR DESCRIPTION
**Pull request by reachjainrahul**
Changes include 
- Kustomization version upgrade to 4.4.1
- scripts for generating manifests files

### CODE REVIEW

File: hack/generate-manifest.sh [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+set -eo pipefail
+
+function echoerr {
+    >&2 echo "$@"
+}
+
+_usage="Usage: $0 [--help|-h]
+Generate a YAML manifest for Nephe using Kustomize and print it to stdout.
+        --help    | -h                      Print this message and exit
+Environment variables IMG_NAME and IMG_TAG must be set when release mode is enabled.
```
Comment (Anandkumar26) [inline hack/generate-manifest.sh:26]: nit: 
```suggestion
Environment variables IMG_NAME and IMG_TAG must be set.
```

**Review by Anandkumar26 — APPROVED** [review]
Minor comment. Can be addressed in a follow up patch.

File: hack/generate-manifest.sh (hunk 1 above)
Comment (reachjainrahul) [reply hack/generate-manifest.sh:26]: Ok.

**Pull request merged**